libmscore/style.cpp

```diff
@@ -269,10 +269,10 @@ static const StyleType styleTypes[] {
       { Sid::harmonyPlacement,         "harmonyPlacement",       int(Placement::ABOVE) },
 
       { Sid::chordSymbolAPosAbove,      "chordSymbolPosAbove",       QPointF(.0, -2.5) },
-      { Sid::chordSymbolAPosBelow,      "chordSymbolPosBelow",       QPointF(.0, 3.5) },
+      { Sid::chordSymbolAPosBelow,      "chordSymbolPosBelow",       QPointF(.0, 7.0) },
```

Comment:
I thing the old values are OK here

Reply:
I was considering leaving the old value, but the only thing was that this offset is calculated from the top of the staff so 3.5 places the chord symbol inside the staff. Let me know if you have any further reasoning for this though! Thank you for the feedback.

Reply:
of course the default setting should make the chord symbol show below staff.

Reply:
I'll take that meaning that it's okay to keep the value :) The old value works with autoplace on of course, but even then if you want to shift it a bit, you still have to move it past 6.5 for it to change so I think this would still be useful when autoplace is on.